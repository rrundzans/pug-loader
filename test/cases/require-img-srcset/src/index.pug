p Hello World!

img(src=require('./image.jpeg')).a
img(srcset=`${require('./image1.jpeg')} 320w, ${require('./image2.jpeg')} 640w` src=require('./image.jpeg')).b