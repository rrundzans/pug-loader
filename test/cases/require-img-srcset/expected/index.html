<p>Hello World!</p><img class="a" src="/assets/images/image.f78b30f4.jpeg"><img class="b" srcset="/assets/images/image1.f78b30f4.jpeg 320w, /assets/images/image2.f78b30f4.jpeg 640w" src="/assets/images/image.f78b30f4.jpeg">